src/main/scala/lc/Main.scala

```diff
@@ -175,11 +169,10 @@ class Server(port: Int){
 
 object LCFramework{
   def main(args: scala.Array[String]) {
-  	val captcha = new Captcha()
+  	val captcha = new Captcha(50)
     val server = new Server(8888)
     server.start()
-    //captcha.beginThread(2)
-    
+    captcha.beginThread(2)
```

Comment:
Minor thing, but wouldn't it be better to start the captcha creation thread before the HTTP server?

Reply:
Since the captcha creation thread in non priority thread. I thought it was fine with starting the server earlier. 
On the other hand, if the server is starting for the first time (new database), i think it would be better to start the captcha creation thread first. 
What do you think is better?

Reply:
Would be better to start the creation thread first.

Reply:
Okay, will make the changes.